Build failure: vimPlugins.fzf-lua

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

Build `vimPlugins.fzf-lua` on `aarch64-linux`
```
nix build ".#legacyPackages.aarch64-linux.vimPlugins.fzf-lua"
```

### Can Hydra reproduce this build failure?

Hydra doesn’t try to build the package.

### Link to Hydra build job

_No response_

### Relevant log output

```console
structuredAttrs is enabled
Sourcing luarocks-move-data-hook.sh
Using luarocksMoveDataHook
Sourcing vim-gen-doc-hook
Sourcing vim-command-check-hook.sh
Using vimCommandCheckHook
Sourcing neovim-require-check-hook.sh
Using neovimRequireCheckHook
Sourcing luarocks-check-hook.sh
Sourcing vim-command-check-hook.sh
Using vimCommandCheckHook
Sourcing neovim-require-check-hook.sh
Using neovimRequireCheckHook
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/ds4pv6j8lwvcv44x20qs50wx8g82rb8v-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Running phase: checkPhase
@nix { "action": "setPhase", "phase": "checkPhase" }
for nvim_exec in nvim; do \
        printf "\n======\n\n" ; \
        $nvim_exec --version | head -n 1 && echo '' ; \
        $nvim_exec --headless --noplugin -u ./scripts/minimal_init.lua \
                -l ./scripts/make_cli.lua ; \
done

======

NVIM v0.11.3

Total number of cases: 102
Total number of groups: 13

tests/actions_spec.lua: x
tests/api_spec.lua: oooooooooooooooooooooo
tests/cache_spec.lua: ooooooooo
tests/devicons_spec.lua: oooooooo
tests/files_spec.lua: ooooooxoooooox
tests/grep_spec.lua: xx
tests/headless_spec.lua: oooo
tests/init_spec.lua: oo
tests/libuv_spec.lua: oooooooo
tests/minicons_spec.lua: ooooo
tests/path_spec.lua: ooooooooooooooo
tests/utils_spec.lua: ooooo
tests/win_spec.lua: oooxoxo

Fails (7) and Notes (0)
FAIL in tests/actions_spec.lua | actions | ui don't freeze on error:
  Failed expectation for screenshot equality to reference at "tests/screenshots/tests-actions_spec.lua---actions---ui-don't-freeze-on-error".
  Different `text` cell at line 4 column 11. Reference: "u". Observed: " ".
  Reference:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭─────────────────────────────────────────────────╮      
  04|~      │> ui should not freeze on action error       0/2 │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │                                                 │      
  07|~      │                                                 │      
  08|~      │                                                 │      
  09|~      │                                                 │      
  10|~      │                                                 │      
  11|~      │                                                 │      
  12|~      │                                                 │      
  13|~      │                                                 │      
  14|~      │                                                 │      
  15|~      │                                                 │      
  16|~      │                                                 │      
  17|~      │                                                 │      
  18|~      │                                                 │      
  19|~      │                                                 │      
  20|~      │                                                 │      
  21|~      │                                                 │      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,3           All 
  
  Observed:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭─────────────────────────────────────────────────╮      
  04|~      │>                                            2/2 │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │▌ aaa                                            │      
  07|~      │  bbb                                            │      
  08|~      │                                                 │      
  09|~      │                                                 │      
  10|~      │                                                 │      
  11|~      │                                                 │      
  12|~      │                                                 │      
  13|~      │                                                 │      
  14|~      │                                                 │      
  15|~      │                                                 │      
  16|~      │                                                 │      
  17|~      │                                                 │      
  18|~      │                                                 │      
  19|~      │                                                 │      
  20|~      │                                                 │      
  21|~      │                                                 │      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,0-1         All 
  Traceback:
    /build/source/lua/fzf-lua/test/screenshot.lua:210
    /build/source/lua/fzf-lua/test/helpers.lua:268
    tests/actions_spec.lua:28

FAIL in tests/files_spec.lua | files | previewer | builtin | 1 + args { true, "ci" }:
  Failed expectation for screenshot equality to reference at "tests/screenshots/tests-files_spec.lua---files---previewer---builtin---1-+-args-{-true,-'ci'-}".
  Different `text` cell at line 14 column 30. Reference: "L". Observed: "─".
  Reference:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭───────────────────── Files ─────────────────────╮      
  04|~      │>                                      84/84 (0) │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │  [DEBUG] [mt] rg --files --sort=path -g !test··││      
  07|~      │▌ <U+E60A> LICENSE                                      │      
  08|~      │  <U+F1064> Makefile                                     │      
  09|~      │  <U+F0354> OPTIONS.md                                   │      
  10|~      │  <U+F0354> README-Win.md                                │      
  11|~      │  <U+F4ED> README.md                                    │      
  12|~      │  <U+E7C5> autoload/fzf_lua.vim                         │      
  13|~      ╰─────────────────────────────────────────────────╯      
  14|~      ╭─────────────────────LICENSE─────────────────────╮      
  15|~      │  1 SELECTED FILE: LICENSE                       │      
  16|~      │                                                 │      
  17|~      │                                                 │      
  18|~      │                                                 │      
  19|~      │                                                 │      
  20|~      │                                                 │      
  21|~      │                                                 │      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,3           All 
  
  Observed:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭───────────────────── Files ─────────────────────╮      
  04|~      │>                                      84/84 (0) │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │  [DEBUG] [mt] rg --files --sort=path -g !test··││      
  07|~      │▌ <U+E60A> LICENSE                                      │      
  08|~      │  <U+F1064> Makefile                                     │      
  09|~      │  <U+F0354> OPTIONS.md                                   │      
  10|~      │  <U+F0354> README-Win.md                                │      
  11|~      │  <U+F4ED> README.md                                    │      
  12|~      │  <U+E7C5> autoload/fzf_lua.vim                         │      
  13|~      ╰─────────────────────────────────────────────────╯      
  14|~      ╭───────────────────────**────────────────────────╮      
  15|~      │  1 SELECTED FILE: **                            │      
  16|~      │                                                 │      
  17|~      │                                                 │      
  18|~      │                                                 │      
  19|~      │                                                 │      
  20|~      │                                                 │      
  21|~      │                                                 │      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,3           All 
  Traceback:
    /build/source/lua/fzf-lua/test/screenshot.lua:210
    /build/source/lua/fzf-lua/test/helpers.lua:268
    /build/source/lua/fzf-lua/test/helpers.lua:483
    tests/files_spec.lua:71

FAIL in tests/files_spec.lua | files | preview should work after chdir #1864:
  Failed expectation for screenshot equality to reference at "tests/screenshots/tests-files_spec.lua---files---preview-should-work-after-chdir-#1864".
  Different `text` cell at line 14 column 28. Reference: " ". Observed: "─".
  Reference:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭───────────────────── Files ─────────────────────╮      
  04|~      │>                                      83/83 (0) │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │  LICENSE                                       ││      
  07|~      │▌ Makefile                                       │      
  08|~      │  OPTIONS.md                                     │      
  09|~      │  README-Win.md                                  │      
  10|~      │  README.md                                      │      
  11|~      │  autoload/fzf_lua.vim                           │      
  12|~      │  autoload/health/fzf_lua.vim                    │      
  13|~      ╰─────────────────────────────────────────────────╯      
  14|~      ╭─────────────────── Makefile ────────────────────╮      
  15|~      │  1 nvim ?= nvim                                        
  16|~      │  2                                              │      
  17|~      │  3 #                                            │      
  18|~      │  4 # Run all tests or specic module tests       │      
  19|~      │  5 #                                            │      
  20|~      │  6 # Test both stable and nightly (assuming `nv`│      
  21|~      │  7 # `make test nvim=nv` or `make test nvim="nvi│      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,3           All 
  
  Observed:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭───────────────────── Files ─────────────────────╮      
  04|~      │>                                      83/83 (0) │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │  LICENSE                                       ││      
  07|~      │▌ Makefile                                       │      
  08|~      │  OPTIONS.md                                     │      
  09|~      │  README-Win.md                                  │      
  10|~      │  README.md                                      │      
  11|~      │  autoload/fzf_lua.vim                           │      
  12|~      │  autoload/health/fzf_lua.vim                    │      
  13|~      ╰─────────────────────────────────────────────────╯      
  14|~      ╭──────────────────── LICENSE ────────────────────╮      
  15|~      │  1 MIT License                                         
  16|~      │  2                                                     
  17|~      │  3 Copyright (c) 2021 iBhagwan                         
  18|~      │  4                                              │      
  19|~      │  5 Permission is hereby granted, free of charge,│      
  20|~      │  6 of this software and associated documentation│      
  21|~      │  7 in the Software without restriction, includin│      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,3           All 
  Traceback:
    /build/source/lua/fzf-lua/test/screenshot.lua:210
    /build/source/lua/fzf-lua/test/helpers.lua:268
    /build/source/lua/fzf-lua/test/helpers.lua:483
    tests/files_spec.lua:174

FAIL in tests/grep_spec.lua | grep | search | regex | 1 + args { false }:
  Failed expectation for screenshot equality to reference at "tests/screenshots/tests-grep_spec.lua---grep---search---regex---1-+-args-{-false-}".
  Different `text` cell at line 14 column 19. Reference: "─". Observed: " ".
  Reference:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭───────────────────── Grep ──────────────────────╮      
  04|~      │(~'"\/$?'`*&&||;[]<>) --tests/grep* >    2/2 (0) │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │  :: <ctrl-g> to Regex Search, Grep string: \(·· │      
  07|~      │  [DEBUG] [st] rg --column --line-number --no-he │      
  08|~      │  ↳ ading --color=always --smart-case --max-colu │      
  09|~      │  ↳ mns=4096  --iglob 'tests/grep*'  -e '\(\~'\' │      
  10|~      │  ↳ '"\\/\$\?'\''`\*&&\|\|;\[]<>\)'              │      
  11|~      │▌ <U+E620> tests/grep_spec.lua:32:21:      search = [== │      
  12|~      │▌ ↳ =[(~'"\/$?'`*&&||;[]<>) --tests/grep*]===],  │      
  13|~      ╰─────────────────────────────────────────────────╯      
  14|~      ╭───────────────── grep_spec.lua ─────────────────╮      
  15|~      │ 29     helpers.FzfLua.grep(child, {             │      
  16|~      │ 30       __expect_lines = true,                 │      
  17|~      │ 31       __screen_opts = screen_opts,           │      
  18|~      │ 32       search = [===[(~'"\/$?'`*&&||;[]<>) --t│      
  19|~      │ 33       debug = 1,                             │      
  20|~      │ 34       silent = true,                         │      
  21|~      │ 35       hidden = false,                        │      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,39          All 
  
  Observed:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭───────────────────── Grep ──────────────────────╮      
  04|~      │(~'"\/$?'`*&&||;[]<>) --tests/grep* >    2/2 (0) │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │  :: <ctrl-g> to Regex Search, Grep string: \(·· │      
  07|~      │  [DEBUG] [st] rg --column --line-number --no-he │      
  08|~      │  ↳ ading --color=always --smart-case --max-colu │      
  09|~      │  ↳ mns=4096  --iglob 'tests/grep*'  -e '\(\~'\' │      
  10|~      │  ↳ '"\\/\$\?'\''`\*&&\|\|;\[]<>\)'              │      
  11|~      │▌ <U+E620> tests/grep_spec.lua:32:21:      search = [== │      
  12|~      │▌ ↳ =[(~'"\/$?'`*&&||;[]<>) --tests/grep*]===],  │      
  13|~      ╰─────────────────────────────────────────────────╯      
  14|~      ╭────────── \$\?'\''`\*&&\|\|;\[]<>\)'  ──────────╮      
  15|~      │  1 Unable to stat file /build/source/[DEBUG] [st│      
  16|~      │                                                 │      
  17|~      │                                                 │      
  18|~      │                                                 │      
  19|~      │                                                 │      
  20|~      │                                                 │      
  21|~      │                                                 │      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,39          All 
  Traceback:
    /build/source/lua/fzf-lua/test/screenshot.lua:210
    /build/source/lua/fzf-lua/test/helpers.lua:268
    /build/source/lua/fzf-lua/test/helpers.lua:483
    tests/grep_spec.lua:29

FAIL in tests/grep_spec.lua | grep | search | regex | 1 + args { true }:
  Failed expectation for screenshot equality to reference at "tests/screenshots/tests-grep_spec.lua---grep---search---regex---1-+-args-{-true-}".
  Different `text` cell at line 14 column 19. Reference: "─". Observed: " ".
  Reference:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭───────────────────── Grep ──────────────────────╮      
  04|~      │(~'"\/$?'`*&&||;[]<>) --tests/grep* >    2/2 (0) │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │  :: <ctrl-g> to Regex Search, Grep string: \(·· │      
  07|~      │  [DEBUG] [mt] rg --column --line-number --no-he │      
  08|~      │  ↳ ading --color=always --smart-case --max-colu │      
  09|~      │  ↳ mns=4096  --iglob 'tests/grep*'  -e '\(\~'\' │      
  10|~      │  ↳ '"\\/\$\?'\''`\*&&\|\|;\[]<>\)'              │      
  11|~      │▌ <U+E620> tests/grep_spec.lua:32:21:      search = [== │      
  12|~      │▌ ↳ =[(~'"\/$?'`*&&||;[]<>) --tests/grep*]===],  │      
  13|~      ╰─────────────────────────────────────────────────╯      
  14|~      ╭───────────────── grep_spec.lua ─────────────────╮      
  15|~      │ 29     helpers.FzfLua.grep(child, {             │      
  16|~      │ 30       __expect_lines = true,                 │      
  17|~      │ 31       __screen_opts = screen_opts,           │      
  18|~      │ 32       search = [===[(~'"\/$?'`*&&||;[]<>) --t│      
  19|~      │ 33       debug = 1,                             │      
  20|~      │ 34       silent = true,                         │      
  21|~      │ 35       hidden = false,                        │      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,39          All 
  
  Observed:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭───────────────────── Grep ──────────────────────╮      
  04|~      │(~'"\/$?'`*&&||;[]<>) --tests/grep* >    2/2 (0) │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │  :: <ctrl-g> to Regex Search, Grep string: \(·· │      
  07|~      │  [DEBUG] [mt] rg --column --line-number --no-he │      
  08|~      │  ↳ ading --color=always --smart-case --max-colu │      
  09|~      │  ↳ mns=4096  --iglob 'tests/grep*'  -e '\(\~'\' │      
  10|~      │  ↳ '"\\/\$\?'\''`\*&&\|\|;\[]<>\)'              │      
  11|~      │▌ <U+E620> tests/grep_spec.lua:32:21:      search = [== │      
  12|~      │▌ ↳ =[(~'"\/$?'`*&&||;[]<>) --tests/grep*]===],  │      
  13|~      ╰─────────────────────────────────────────────────╯      
  14|~      ╭────────── \$\?'\''`\*&&\|\|;\[]<>\)'  ──────────╮      
  15|~      │  1 Unable to stat file /build/source/[DEBUG] [mt│      
  16|~      │                                                 │      
  17|~      │                                                 │      
  18|~      │                                                 │      
  19|~      │                                                 │      
  20|~      │                                                 │      
  21|~      │                                                 │      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,39          All 
  Traceback:
    /build/source/lua/fzf-lua/test/screenshot.lua:210
    /build/source/lua/fzf-lua/test/helpers.lua:268
    /build/source/lua/fzf-lua/test/helpers.lua:483
    tests/grep_spec.lua:29

FAIL in tests/win_spec.lua | win | hide | actions on multi-select but zero-match #1961: /build/source/lua/fzf-lua/test/helpers.lua:323: Timed out waiting for condition after 2000 ms
  
  
  
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭─────────────────── Files  h ────────────────────╮      
  04|~      │/build/source/README.mda-non-exist-file   0/16.. │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │                                                 │      
  07|~      │                                                 │      
  08|~      │                                                 │      
  09|~      │                                                 │      
  10|~      │                                                 │      
  11|~      │                                                 │      
  12|~      │                                                 │      
  13|~      ╰─────────────────────────────────────────────────╯      
  14|~      ╭─────────────────────────────────────────────────╮      
  15|~      │                                                 │      
  16|~      │                                                 │      
  17|~      │                                                 │      
  18|~      │                                                 │      
  19|~      │                                                 │      
  20|~      │                                                 │      
  21|~      │                                                 │      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,40          All 
  
  --|---------|---------|---------|---------|---------|---------|----
  01|0000000000000000000000000000000000000000000000000000000000000000
  02|1111111111111111111111111111111111111111111111111111111111111111
  03|1111111222222222222222222222222222333222222222222222222222111111
  04|1111111244444444444444555555555555555555555555522266666622111111
  05|1111111277777777777777777777777777777777777777777777777722111111
  06|1111111222222222222222222222222222222222222222222222222222111111
  07|1111111222222222222222222222222222222222222222222222222222111111
  08|1111111222222222222222222222222222222222222222222222222222111111
  09|1111111222222222222222222222222222222222222222222222222222111111
  10|1111111222222222222222222222222222222222222222222222222222111111
  11|1111111222222222222222222222222222222222222222222222222222111111
  12|1111111222222222222222222222222222222222222222222222222222111111
  13|1111111222222222222222222222222222222222222222222222222222111111
  14|1111111222222222222222222222222222222222222222222222222222111111
  15|1111111222222222222222222222222222222222222222222222222222111111
  16|1111111288888888888888888888888888888888888888888888888882111111
  17|1111111288888888888888888888888888888888888888888888888882111111
  18|1111111288888888888888888888888888888888888888888888888882111111
  19|1111111288888888888888888888888888888888888888888888888882111111
  20|1111111288888888888888888888888888888888888888888888888882111111
  21|1111111288888888888888888888888888888888888888888888888882111111
  22|1111111222222222222222222222222222222222222222222222222222111111
  23|1111111111111111111111111111111111111111111111111111111111111111
  24|9999999999999999999999999999999999999999999999999999999999999999
  25|0000000000000000000000000000000000000000000000000000000000000000
  26|0000000000000000000000000000000000000000000000000000000000000000
  27|0000000000000000000000000000000000000000000000000000000000000000
  28|::::::::::::::;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
  Traceback:
    /build/source/lua/fzf-lua/test/helpers.lua:323
    tests/win_spec.lua:140

FAIL in tests/win_spec.lua | win | previewer | split flex layout resize:
  Failed expectation for screenshot equality to reference at "tests/screenshots/tests-win_spec.lua---win---previewer---split-flex-layout-resize-002".
  Different `text` cell at line 1 column 23. Reference: "3". Observed: " ".
  Reference:
  --|---------|---------|---------|---------|---------|---------|-----
  01|>                     3/3 ╭─────────────────foo─────────────────╮
  02|───────────────────────── │  1 SELECTED FILE: foo               │
  03|▌ foo                     │                                     │
  04|  bar                     │                                     │
  05|  baz                     │                                     │
  06|                          │                                     │
  07|                          │                                     │
  08|                          │                                     │
  09|                          │                                     │
  10|                          │                                     │
  11|                          │                                     │
  12|                          │                                     │
  13|                          │                                     │
  14|                          │                                     │
  15|                          │                                     │
  16|                          │                                     │
  17|                          │                                     │
  18|                          │                                     │
  19|                          │                                     │
  20|                          │                                     │
  21|                          │                                     │
  22|                          │                                     │
  23|                          ╰─────────────────────────────────────╯
  24|<d/bhagwan/Sources/nvim/fzf-lua//27829:/bin/sh 1,3            All
  25|                                                                 
  26|                                                                 
  27|                                                                 
  28|-- TERMINAL --                                                   
  
  Observed:
  --|---------|---------|---------|---------|---------|---------|-----
  01|>                         ╭─────────────────foo─────────────────╮
  02|──────────────────────────│  1 SELECTED FILE: foo               │
  03|▌ foo                     │                                     │
  04|  bar                     │                                     │
  05|  baz                     │                                     │
  06|                          │                                     │
  07|                          │                                     │
  08|                          │                                     │
  09|                          │                                     │
  10|                          │                                     │
  11|                          │                                     │
  12|                          │                                     │
  13|                          │                                     │
  14|╭─────────────────────────│                                     │
  15|│                         │                                     │
  16|│                         │                                     │
  17|│                         │                                     │
  18|│                         │                                     │
  19|│                         │                                     │
  20|│                         │                                     │
  21|│                         │                                     │
  22|│                         │                                     │
  23|╰─────────────────────────╰─────────────────────────────────────╯
  24|<hywa0i56d4a90fs77dj6im21jkf-bash-5.3p3/bin/sh 1,3            All
  25|                                                                 
  26|                                                                 
  27|                                                                 
  28|-- TERMINAL --                                                   
  Traceback:
    /build/source/lua/fzf-lua/test/screenshot.lua:210
    /build/source/lua/fzf-lua/test/helpers.lua:268
    tests/win_spec.lua:202

make: *** [Makefile:15: test] Error 1
```

### Additional context

I don't know the hydra status, the search kept timing out with a 500 error.

I'm emulating `aarch64` with `binfmt.emulatedSystems` to build.

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.16.1-zen1, NixOS, 25.11 (Xantusia), 25.11.20250819.2007595`
 - multi-user?: `yes`
 - sandbox: `relaxed`
 - version: `nix-env (Nix) 2.28.4`
 - channels(lunik1): `""`
 - nixpkgs: `/nix/store/pxnx7dbr0pz3qpgw2r315wdbb00afdw7-source`

### Notify maintainers

@mrcjkb 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

Hmm I still fail tests when building on master
```
structuredAttrs is enabled
Sourcing luarocks-check-hook.sh
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/ds4pv6j8lwvcv44x20qs50wx8g82rb8v-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Running phase: checkPhase
@nix { "action": "setPhase", "phase": "checkPhase" }
for nvim_exec in nvim; do \
        printf "\n======\n\n" ; \
        $nvim_exec --version | head -n 1 && echo '' ; \
        $nvim_exec --headless --noplugin -u ./scripts/minimal_init.lua \
                -l ./scripts/make_cli.lua ; \
done

======

NVIM v0.11.4

Total number of cases: 102
Total number of groups: 13

tests/actions_spec.lua: x
tests/api_spec.lua: oooooooooooooooooooooo
tests/cache_spec.lua: ooooooooo
tests/devicons_spec.lua: oooooooo
tests/files_spec.lua: ooooooxoooooox
tests/grep_spec.lua: xo
tests/headless_spec.lua: oooo
tests/init_spec.lua: oo
tests/libuv_spec.lua: oooooooo
tests/minicons_spec.lua: ooooo
tests/path_spec.lua: ooooooooooooooo
tests/utils_spec.lua: ooooo
tests/win_spec.lua: oooooxo

Fails (5) and Notes (0)
FAIL in tests/actions_spec.lua | actions | ui don't freeze on error:
  Failed expectation for screenshot equality to reference at "tests/screenshots/tests-actions_spec.lua---actions---ui-don't-freeze-on-error".
  Different `text` cell at line 4 column 11. Reference: "u". Observed: " ".
  Reference:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭─────────────────────────────────────────────────╮      
  04|~      │> ui should not freeze on action error       0/2 │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │                                                 │      
  07|~      │                                                 │      
  08|~      │                                                 │      
  09|~      │                                                 │      
  10|~      │                                                 │      
  11|~      │                                                 │      
  12|~      │                                                 │      
  13|~      │                                                 │      
  14|~      │                                                 │      
  15|~      │                                                 │      
  16|~      │                                                 │      
  17|~      │                                                 │      
  18|~      │                                                 │      
  19|~      │                                                 │      
  20|~      │                                                 │      
  21|~      │                                                 │      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,3           All 
  
  Observed:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭─────────────────────────────────────────────────╮      
  04|~      │>                                            2/2 │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │▌ aaa                                            │      
  07|~      │  bbb                                            │      
  08|~      │                                                 │      
  09|~      │                                                 │      
  10|~      │                                                 │      
  11|~      │                                                 │      
  12|~      │                                                 │      
  13|~      │                                                 │      
  14|~      │                                                 │      
  15|~      │                                                 │      
  16|~      │                                                 │      
  17|~      │                                                 │      
  18|~      │                                                 │      
  19|~      │                                                 │      
  20|~      │                                                 │      
  21|~      │                                                 │      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,0-1         All 
  Traceback:
    /build/source/lua/fzf-lua/test/screenshot.lua:210
    /build/source/lua/fzf-lua/test/helpers.lua:268
    tests/actions_spec.lua:28

FAIL in tests/files_spec.lua | files | previewer | builtin | 1 + args { true, "ci" }:
  Failed expectation for screenshot equality to reference at "tests/screenshots/tests-files_spec.lua---files---previewer---builtin---1-+-args-{-true,-'ci'-}".
  Different `text` cell at line 14 column 30. Reference: "L". Observed: "─".
  Reference:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭───────────────────── Files ─────────────────────╮      
  04|~      │>                                      84/84 (0) │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │  [DEBUG] [mt] rg --files --sort=path -g !test··││      
  07|~      │▌ <U+E60A> LICENSE                                      │      
  08|~      │  <U+F1064> Makefile                                     │      
  09|~      │  <U+F0354> OPTIONS.md                                   │      
  10|~      │  <U+F0354> README-Win.md                                │      
  11|~      │  <U+F4ED> README.md                                    │      
  12|~      │  <U+E7C5> autoload/fzf_lua.vim                         │      
  13|~      ╰─────────────────────────────────────────────────╯      
  14|~      ╭─────────────────────LICENSE─────────────────────╮      
  15|~      │  1 SELECTED FILE: LICENSE                       │      
  16|~      │                                                 │      
  17|~      │                                                 │      
  18|~      │                                                 │      
  19|~      │                                                 │      
  20|~      │                                                 │      
  21|~      │                                                 │      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,3           All 
  
  Observed:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭───────────────────── Files ─────────────────────╮      
  04|~      │>                                      84/84 (0) │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │  [DEBUG] [mt] rg --files --sort=path -g !test··││      
  07|~      │▌ <U+E60A> LICENSE                                      │      
  08|~      │  <U+F1064> Makefile                                     │      
  09|~      │  <U+F0354> OPTIONS.md                                   │      
  10|~      │  <U+F0354> README-Win.md                                │      
  11|~      │  <U+F4ED> README.md                                    │      
  12|~      │  <U+E7C5> autoload/fzf_lua.vim                         │      
  13|~      ╰─────────────────────────────────────────────────╯      
  14|~      ╭───────────────────────**────────────────────────╮      
  15|~      │  1 SELECTED FILE: **                            │      
  16|~      │                                                 │      
  17|~      │                                                 │      
  18|~      │                                                 │      
  19|~      │                                                 │      
  20|~      │                                                 │      
  21|~      │                                                 │      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,3           All 
  Traceback:
    /build/source/lua/fzf-lua/test/screenshot.lua:210
    /build/source/lua/fzf-lua/test/helpers.lua:268
    /build/source/lua/fzf-lua/test/helpers.lua:483
    tests/files_spec.lua:71

FAIL in tests/files_spec.lua | files | preview should work after chdir #1864:
  Failed expectation for screenshot equality to reference at "tests/screenshots/tests-files_spec.lua---files---preview-should-work-after-chdir-#1864".
  Different `text` cell at line 14 column 28. Reference: " ". Observed: "─".
  Reference:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭───────────────────── Files ─────────────────────╮      
  04|~      │>                                      83/83 (0) │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │  LICENSE                                       ││      
  07|~      │▌ Makefile                                       │      
  08|~      │  OPTIONS.md                                     │      
  09|~      │  README-Win.md                                  │      
  10|~      │  README.md                                      │      
  11|~      │  autoload/fzf_lua.vim                           │      
  12|~      │  autoload/health/fzf_lua.vim                    │      
  13|~      ╰─────────────────────────────────────────────────╯      
  14|~      ╭─────────────────── Makefile ────────────────────╮      
  15|~      │  1 nvim ?= nvim                                        
  16|~      │  2                                              │      
  17|~      │  3 #                                            │      
  18|~      │  4 # Run all tests or specic module tests       │      
  19|~      │  5 #                                            │      
  20|~      │  6 # Test both stable and nightly (assuming `nv`│      
  21|~      │  7 # `make test nvim=nv` or `make test nvim="nvi│      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,3           All 
  
  Observed:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭───────────────────── Files ─────────────────────╮      
  04|~      │>                                      83/83 (0) │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │  LICENSE                                       ││      
  07|~      │▌ Makefile                                       │      
  08|~      │  OPTIONS.md                                     │      
  09|~      │  README-Win.md                                  │      
  10|~      │  README.md                                      │      
  11|~      │  autoload/fzf_lua.vim                           │      
  12|~      │  autoload/health/fzf_lua.vim                    │      
  13|~      ╰─────────────────────────────────────────────────╯      
  14|~      ╭──────────────────── LICENSE ────────────────────╮      
  15|~      │  1 MIT License                                         
  16|~      │  2                                                     
  17|~      │  3 Copyright (c) 2021 iBhagwan                         
  18|~      │  4                                              │      
  19|~      │  5 Permission is hereby granted, free of charge,│      
  20|~      │  6 of this software and associated documentation│      
  21|~      │  7 in the Software without restriction, includin│      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,3           All 
  Traceback:
    /build/source/lua/fzf-lua/test/screenshot.lua:210
    /build/source/lua/fzf-lua/test/helpers.lua:268
    /build/source/lua/fzf-lua/test/helpers.lua:483
    tests/files_spec.lua:174

FAIL in tests/grep_spec.lua | grep | search | regex | 1 + args { false }:
  Failed expectation for screenshot equality to reference at "tests/screenshots/tests-grep_spec.lua---grep---search---regex---1-+-args-{-false-}".
  Different `text` cell at line 14 column 19. Reference: "─". Observed: " ".
  Reference:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭───────────────────── Grep ──────────────────────╮      
  04|~      │(~'"\/$?'`*&&||;[]<>) --tests/grep* >    2/2 (0) │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │  :: <ctrl-g> to Regex Search, Grep string: \(·· │      
  07|~      │  [DEBUG] [st] rg --column --line-number --no-he │      
  08|~      │  ↳ ading --color=always --smart-case --max-colu │      
  09|~      │  ↳ mns=4096  --iglob 'tests/grep*'  -e '\(\~'\' │      
  10|~      │  ↳ '"\\/\$\?'\''`\*&&\|\|;\[]<>\)'              │      
  11|~      │▌ <U+E620> tests/grep_spec.lua:32:21:      search = [== │      
  12|~      │▌ ↳ =[(~'"\/$?'`*&&||;[]<>) --tests/grep*]===],  │      
  13|~      ╰─────────────────────────────────────────────────╯      
  14|~      ╭───────────────── grep_spec.lua ─────────────────╮      
  15|~      │ 29     helpers.FzfLua.grep(child, {             │      
  16|~      │ 30       __expect_lines = true,                 │      
  17|~      │ 31       __screen_opts = screen_opts,           │      
  18|~      │ 32       search = [===[(~'"\/$?'`*&&||;[]<>) --t│      
  19|~      │ 33       debug = 1,                             │      
  20|~      │ 34       silent = true,                         │      
  21|~      │ 35       hidden = false,                        │      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,39          All 
  
  Observed:
  --|---------|---------|---------|---------|---------|---------|----
  01|                                                                
  02|~                                                               
  03|~      ╭───────────────────── Grep ──────────────────────╮      
  04|~      │(~'"\/$?'`*&&||;[]<>) --tests/grep* >    2/2 (0) │      
  05|~      │──────────────────────────────────────────────── │      
  06|~      │  :: <ctrl-g> to Regex Search, Grep string: \(·· │      
  07|~      │  [DEBUG] [st] rg --column --line-number --no-he │      
  08|~      │  ↳ ading --color=always --smart-case --max-colu │      
  09|~      │  ↳ mns=4096  --iglob 'tests/grep*'  -e '\(\~'\' │      
  10|~      │  ↳ '"\\/\$\?'\''`\*&&\|\|;\[]<>\)'              │      
  11|~      │▌ <U+E620> tests/grep_spec.lua:32:21:      search = [== │      
  12|~      │▌ ↳ =[(~'"\/$?'`*&&||;[]<>) --tests/grep*]===],  │      
  13|~      ╰─────────────────────────────────────────────────╯      
  14|~      ╭────────── \$\?'\''`\*&&\|\|;\[]<>\)'  ──────────╮      
  15|~      │  1 Unable to stat file /build/source/[DEBUG] [st│      
  16|~      │                                                 │      
  17|~      │                                                 │      
  18|~      │                                                 │      
  19|~      │                                                 │      
  20|~      │                                                 │      
  21|~      │                                                 │      
  22|~      ╰─────────────────────────────────────────────────╯      
  23|~                                                               
  24|[No Name]                                     0,0-1          All
  25|                                                                
  26|                                                                
  27|                                                                
  28|-- TERMINAL --                                1,39          All 
  Traceback:
    /build/source/lua/fzf-lua/test/screenshot.lua:210
    /build/source/lua/fzf-lua/test/helpers.lua:268
    /build/source/lua/fzf-lua/test/helpers.lua:483
    tests/grep_spec.lua:29

FAIL in tests/win_spec.lua | win | previewer | split flex layout resize:
  Failed expectation for screenshot equality to reference at "tests/screenshots/tests-win_spec.lua---win---previewer---split-flex-layout-resize-002".
  Different `text` cell at line 1 column 23. Reference: "3". Observed: " ".
  Reference:
  --|---------|---------|---------|---------|---------|---------|-----
  01|>                     3/3 ╭─────────────────foo─────────────────╮
  02|───────────────────────── │  1 SELECTED FILE: foo               │
  03|▌ foo                     │                                     │
  04|  bar                     │                                     │
  05|  baz                     │                                     │
  06|                          │                                     │
  07|                          │                                     │
  08|                          │                                     │
  09|                          │                                     │
  10|                          │                                     │
  11|                          │                                     │
  12|                          │                                     │
  13|                          │                                     │
  14|                          │                                     │
  15|                          │                                     │
  16|                          │                                     │
  17|                          │                                     │
  18|                          │                                     │
  19|                          │                                     │
  20|                          │                                     │
  21|                          │                                     │
  22|                          │                                     │
  23|                          ╰─────────────────────────────────────╯
  24|<d/bhagwan/Sources/nvim/fzf-lua//27829:/bin/sh 1,3            All
  25|                                                                 
  26|                                                                 
  27|                                                                 
  28|-- TERMINAL --                                                   
  
  Observed:
  --|---------|---------|---------|---------|---------|---------|-----
  01|>                         ╭─────────────────foo─────────────────╮
  02|──────────────────────────│  1 SELECTED FILE: foo               │
  03|▌ foo                     │                                     │
  04|  bar                     │                                     │
  05|  baz                     │                                     │
  06|                          │                                     │
  07|                          │                                     │
  08|                          │                                     │
  09|                          │                                     │
  10|                          │                                     │
  11|                          │                                     │
  12|                          │                                     │
  13|                          │                                     │
  14|╭─────────────────────────│                                     │
  15|│                         │                                     │
  16|│                         │                                     │
  17|│                         │                                     │
  18|│                         │                                     │
  19|│                         │                                     │
  20|│                         │                                     │
  21|│                         │                                     │
  22|│                         │                                     │
  23|╰─────────────────────────╰─────────────────────────────────────╯
  24|<hywa0i56d4a90fs77dj6im21jkf-bash-5.3p3/bin/sh 1,3            All
  25|                                                                 
  26|                                                                 
  27|                                                                 
  28|-- TERMINAL --                                                   
  Traceback:
    /build/source/lua/fzf-lua/test/screenshot.lua:210
    /build/source/lua/fzf-lua/test/helpers.lua:268
    tests/win_spec.lua:202

make: *** [Makefile:15: test] Error 1
```

--- a/pkgs/development/lua-modules/overrides.nix
+++ b/pkgs/development/lua-modules/overrides.nix
@@ -154,7 +154,8 @@ in
     # FIXME: https://github.com/NixOS/nixpkgs/issues/431458
     # fzf-lua throws `address already in use` on darwin
     # Previewer transient failure
-    doCheck = !stdenv.hostPlatform.isDarwin;
+    # UI tests fail either transiently or consistently in certain software/hardware configurations
+    doCheck = false;
     checkInputs = [
       fd
       fzf

